common-theme/layouts/partials/block/link.html

```diff
@@ -1,6 +1,8 @@
 {{ $blockData := .Page.Scratch.Get "blockData" }}
 {{ $randomImage := (index (resources.Match "custom-images/backgrounds/**.**" | shuffle | first 1) 0) }}
-{{ $randomImage = $randomImage.Resize "480x" }}
+{{ if $randomImage }}
```

Comment:
This is just a little guard against an annoying Hugo server fault that was getting in the way of this fix - it is basically irrelevant otherwise